pkgs/development/python-modules/prometheus_flask_exporter/default.nix

```diff
@@ -0,0 +1,22 @@
+{ stdenv, buildPythonPackage, fetchPypi, flask, prometheus_client }:
+
+buildPythonPackage rec {
+  pname = "prometheus_flask_exporter";
+  version = "0.15.4";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "c590656b45fa6dd23d81dec3d3dc1e31b17fcba48310f69d0ff31b5c865fc799";
+  };
+
+  propagatedBuildInputs = [
+    flask prometheus_client
+  ];
+
```

Comment:
There are no tests included. Please try to checkout from source and check if they have unit tests, and try to run them. Unit tests give a good indication that they package has a high degree of validity and correctness given the python package set.

If tests are not available, then please use `pythonImportsCheck` to import the most important modules. This isn't as good as unit tests, but will usually give a good indication of run-time errors. Please see [pythonImportsCheck documentation](https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#using-pythonimportscheck) for more information.

```

----------------------------------------------------------------------
Ran 0 tests in 0.000s
```